deployments/helm/values.yaml

```diff
@@ -10,6 +10,9 @@ node:
 # Ethereum node URL. A value for this variable is mandatory.
 base_layer_node_url:
 
+# If true, metrics will be collected and exposed on the monitoring port.
+collect_metrics: true
```

Comment:
1. A cluster that runs inside a cluster (papyrus in this case) can't guarantee that its metrics would be collected by some other app.
2. I don't see anywhere in the helm chart which reference this condition.